build/Build.fsproj

```diff
@@ -1,19 +1,17 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
-
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>net6.0</TargetFramework>
   </PropertyGroup>
-
   <ItemGroup>
     <Compile Include="Tools.fs" />
     <Compile Include="Files.fs" />
     <Compile Include="Program.fs" />
   </ItemGroup>
-
   <ItemGroup>
     <PackageReference Include="Fake.Core.Environment" Version="5.20.4" />
     <PackageReference Include="Fake.Core.Target" Version="5.20.4" />
   </ItemGroup>
-
-</Project>
+  <Import Project="..\.paket\Paket.Restore.targets" />
```

Comment:
Same here, build project doesn't depend on paket and doesn't need to share dependencies with other projects

Reply:
Hmm, I think Paket did this automatically when running `dotnet paket install`. I will undo the changes 👍